debian/changelog

```diff
@@ -1,3 +1,23 @@
+tiqit (1.0.6-1) trusty; urgency=low
+
+  * Added missing code for config directories
+
+ -- Jonathan Loh <joloh@ensoft.co.uk> Thurs, 15 Nov 2018 15:30:00 +0100
+
+tiqit (1.0.5-1) trusty; urgency=low
+
+  * Fixes to the make_deb script.
+  * Gracefully handle searches updating between 0 and 1 results.
+  * Sort search fields by display name, rather than internal name.
+  * Stop severity graph overflow.
+  * Reset all forms on the bug page with the reset button.
+  * Don't clear the search box when changing condition from is to is not.
+  * Set attachment titles to filenames by default.
+  * Stop logical operators being changed when removing entries from
+    a compound search.
+
+ -- Jonathan Loh <joloh@ensoft.co.uk> Wed, 4 Jul 2018 15:30:00 +0100
+
```

Comment:
Not necessary to change now, but in future it would be cleaner to only increment the version number once for each PR to master and production release.

Perhaps this got incremented twice because we created a dev build for testing from the development branch. In that scenario it would be better to increament only the DEV_VER field and create and start building up the new the changelog entry with a temporary name of like 1.0.4dev1-1, that would then be renamed to 1.0.5 in the PR to master.

Reply:
Done